brew-start.sh

```diff
@@ -7,7 +7,8 @@
 
 
 #-----BOOTSTRAP HELPERS-----#
-source helpers.sh
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
_Nice trick!_ - I'm definitely going to steal this 😄

Reply:
Our other solution uses some bits that don't work on MacOS and we didn't want to break MacOS compatibility but this should be fine!